epam-vote-system-controller/src/main/java/com/epam/vote/controller/TestController.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.vote.controller;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * @author Dzianis Zatsiupa
```

Comment:
Let's write comment in the next formatting:
/**
 * "Description of class".
 * <p/>
 * Copyright (C) 2021
 * <p/>
 * Date: "CURRENT DATE"
 * @author "YOUR NAME"
 */